### PR TITLE
Add missing endpoints, edit responses

### PR DESCRIPTION
**Pull request by johanohly**
Adds custom fields
Adds webhooks
Adds school
Adds gcode macros
Adds courses
Adds partner

Adds printer > AutoPrint
Adds printer > OneClickPrint
Adds job > GetDetails
Adds printer groups
Adds queue > ReviveItem
Adds queue > groups

Updates permissions scopes
Updates permissions required for each endpoint
Updates a few old responses

### CODE REVIEW

File: source/includes/_printers.md [hunk 1]
```diff
@@ -569,34 +582,378 @@ curl https://api.simplyprint.io/{id}/printers/Delete?pid=1234&just_connection=1
...
 
 ### Response
 
 | Field     | Type    | Description                         |
-| --------- | ------- | ----------------------------------- |
+|-----------|---------|-------------------------------------|
 | `status`  | boolean | True if the request was successful. |
 | `message` | string  | Error message if `status` is false. |
 
+## List one-click printers
```
Comment (AlbertMN) [inline source/includes/_printers.md:607]: @johanohly Must be "1-Click"

File: source/includes/_printers.md [hunk 2]
```diff
@@ -569,34 +582,378 @@ curl https://api.simplyprint.io/{id}/printers/Delete?pid=1234&just_connection=1
...
+  "printers": [
+    {
+      ...
+    }
+  ]
+}
+```
+
+`GET /{id}/printers/autoprint/CheckState`
+
```
Comment (AlbertMN) [inline source/includes/_printers.md:730]: @johanohly missing "Requires Print Farm" notice

File: source/includes/_queue.md [hunk 3]
```diff
@@ -188,6 +204,120 @@ This endpoint gets the next item in the queue for the specified printer. The nex
 | `queue.matches[].printed` | integer   | The amount of completed prints of this item (from print queue). Only present if `match` is true.                                    |
 | `queue.matches[].left`    | integer   | The amount of prints left (from print queue). Only present if `match` is true.                                                      |
 
+## Get queue item
+
```
Comment (AlbertMN) [inline source/includes/_queue.md:208]: Missing "Requires Pro plan" notice

Comment (AlbertMN) [reply source/includes/_queue.md:208]: Or... It can be present later? I need to see a preview of the rendered site I think

Comment (johanohly) [reply source/includes/_queue.md:208]: Totally fair. Not sure there is an easy way to implement public previews, but for now running it manually is sufficient I think.

**Review by AlbertMN — APPROVED** [review]
LGTM

**Pull request merged**